book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def add_rental(person)
+    Rental.new(self, person)
```

Comment:
is it really necessary? Because when the book is rented a intance variable @date is automatically created with Date.now() whic will assign the date rented. My plan was not to ask the user for the date since it's renting now, it's one step less that the user needs to do!

Reply:
* Kindly add a date parameter as pointed out by the previous reviewer, you need to pass a date to successfully create a rental so your method should look like this 
* ` def add_rental(person, date)
    Rental.new(date, self, person)
  end
`